.github/workflows/tests.yml

```diff
@@ -54,7 +54,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, windows-latest ]
-        python-version: ['3.9', '3.10', '3.11']
+        # python-version: ['3.9', '3.10', '3.11']
```

Comment:
Temporary disabling these tests due to incompatibility in vcr stubs, addressing later in https://humansignal.atlassian.net/browse/RND-106

Reply:
can you enable 3.11 so at least some tests can be run?

Reply:
I left [3.11 enabled](https://github.com/HumanSignal/Adala/pull/162/files/d6e5e355877f9688c12587abc6b8f59f42ceffb7#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR58)